.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 0.10
+  - "stable"
 before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
```

Comment:
Seems like this is the line causing it, so remove it?

If `xvfb` is actually needed then we can add it to a `services` array in this file

Reply:
I actually just added that as the second commit to try and fix it....so same error was happening before I added that. The `xvfb` piece was what I was gonna try next based on a stackoverflow i'm looking at